src/components/user/profile-activity-graph.tsx

```diff
@@ -88,7 +88,7 @@ export function ProfileActivityGraph({
               strokeDashoffset: dashOffset,
             }}
           />
-          <InnerCircle r="24" cx="50" cy="50" />
+          <InnerCircle r={(progress / max_level) * 40} cx="50" cy="50" />
```

Comment:
Not sure about this one. 
I'm fine with either size or bar, both are confusing imo.